doc/manual/contracts.md

```diff
@@ -711,34 +717,37 @@ value and continues with the second argument (here, our wrapped `value`).
 
 Let us see if we indeed preserved laziness:
 
-```nickel
-nickel>
-let config | NumberBoolDict = {
-  "1" | std.FailWith "ooch" = null, # same as our previous "fail"
-  "0" | doc "Some information" = true,
-}
-nickel>:query config "0"
+```nickel-repl
+> let config | NumberBoolDict = {
+    "1" | std.FailWith "ooch" = null, # same as our previous "fail"
+    "0" | doc "Some information" = true,
+  }
+
+# Don't parse this in tests hide-line
```

Comment:
What's the purpose of this comment, and where is it handled? I think I missed it somehow... Does it appear in the final manual? Also, I hope we can get rid of it eventually - starting an REPL engine using `ReplImpl` to actually test the output of the REPL examples should be doable.

Reply:
It's a bit convoluted. The code parsing out REPL inputs expects to find a `>` directly following an empty line, so it will skip this particular input because of the comment. The `hide-line` is for gatsby to not display the comment on the website.

Reply:
Ah. I would have suggested to add a comment to explain that, but...that's probably a bad idea :stuck_out_tongue: is the `hide-line` a vanilla Gatsby feature or something we filter out specifically in nickel-lang.org ?

Reply:
The `hide-line` is a feature of the [`gatsby-remark-prismjs` plugin](https://github.com/gatsbyjs/gatsby/blob/master/packages/gatsby-remark-prismjs/README.md#line-hiding).